earth_enterprise/src/fusion/fusionui/RasterAssetWidget.cpp

```diff
@@ -649,6 +649,10 @@ void RasterAssetWidget::AssembleEditRequest(
   } else {
     request->config.provider_id_ = 0;
   }
+
+  if (!acquisition_date_wrapper_->IsValidDate()) {
```

Comment:
We have to make an exception for 0000-00-00. This should still be allowed and represents a blank date. It's just dates, such as 2000-00-00 that shouldn't be allowed.